app/models/assignment.rb

```diff
@@ -280,7 +280,7 @@ def median(marks)
   def self.get_current_assignment
     # start showing (or "featuring") the assignment 3 days before it's due
     # query uses Date.today + 4 because results from db seems to be off by 1
-    current_assignment = Assignment.where('due_date <= ?', Date.today + 4)
+    current_assignment = Assignment.where('due_date <= ?', Date.today + 4).include
```

Comment:
There's a test failing because of this change I think, and this does look kind of weird. Is there supposed to be something after the `.include` that's missing?